media/components/sdfv/util/vscode_renderer.js

```diff
@@ -1,51 +0,0 @@
-// Copyright 2020-2021 ETH Zurich and the DaCe-VSCode authors.
-// All rights reserved.
-
-class VSCodeRenderer extends daceSDFGRenderer {
```

Comment:
@phschaad do we want to keep this code one way or another?

Reply:
I vote yes, I make heavy use of the `VSCodeRenderer` in a currently open PR. I also cleaned up the initialization (see comment above on `index.html`), so I would prefer if we can keep this in - I generally like the minimally-invasive approach of extending the renderer that this enables.